### DISCUSSION TITLE
I'm thinking about making an extension for ads.

### DISCUSSION THREAD

**Discussion by AdamMady** [TurboWarp/extensions#1194] [Polls]
I've been wanting to make an extension where it allows you to display ads using the CrazyGames API, however, I'm not sure if this would just be the worst extension ever, because scratch games have never really been very profitable on any website but at the same time it's not really supposed to be profitable.

obviously you'd need to upload it on crazygames.

if it works out I could try AdSense next.

**Comment by CubesterYT** [comment]
If I remember correctly, I don't think any type of monetization extensions would ever be allowed to be added here.

**Reply by AdamMady** [reply]
where does it say that? genuinely curious.

**Reply by CubesterYT** [reply]
https://github.com/TurboWarp/extensions/issues/933

We don't directly mention ads, but it would fall under a similar, monetary category.

**Reply by AdamMady** [reply]
I see..

Alright thank you!, Hopefully when someone confirms this I'll think about it.

**Reply by Drago-Cuven** [reply]
still this "could" be good. especially for the turbowarpers who actually KNOW how to make a good game. (and besides there are other galleries you could put it in *wink wink*)

**Reply by PURPL3-CAT** [reply]
Would an extension that allows for the implementation of things, one of which could be ads, likely be added?

**Comment by CubesterYT** [comment]
But you can still make one.

**Comment by softedco** [comment]
It already exists, a crazygames sdk extension was made by me some time ago and it didn't get accepted because there was no license information and also because such extensions aren't really tolerated here. #113

**Reply by AdamMady** [reply]
well that's unfortunate, glad I made sure before actually giving it a shot though. in my opinion, I would say some way to monetize scratch games properly would be absolutely revolutionary for scratch, truly a shame it wouldn't be allowed here. thank you for confirming!

**Reply by CubesterYT** [reply]
> well that's unfortunate, glad I made sure before actually giving it a shot though. in my opinion, I would say some way to monetize scratch games properly would be absolutely revolutionary for scratch, truly a shame it wouldn't be allowed here. thank you for confirming!

@AdamMady, again, you can always still use it or make it for yourself.

**Comment by GaypataponALT** [comment]
How about a google play sdk extention? for making mobil games?

**Reply by softedco** [reply]
This won't make it on there for the same reasons

**Reply by unknown07724** [reply]
I am working on 1 but it won't have blocks for ads

**Comment by GaypataponALT** [comment]
You could still make the extention but don't post it to github, why not
post it somewhere else?

**Comment by HirujaEdurapola** [comment]
Create this but don't post it on github. I could really use this so if u do make it can you send me?  videohiruja@gmail.com
